Entity/Dynamic.php

```diff
@@ -170,6 +170,10 @@ public function getData()
      */
     public function __set($name, $value)
     {
+        if ($value instanceof \DateTime) {
```

Comment:
@alexander-schranz Yes, because `date` is defined as `string` in the XML. We could change that, but then we have two different cases -> if property_exists, it's a DateTime object, if not, it's a string...